### DISCUSSION TITLE
Does the Rock64 image include the patch for the 5V errors?

### DISCUSSION THREAD

**Discussion by WizardKnight** [johang/sd-card-images#121] [Q&A]
I was wondering if the Debian 12.5 images included the patch for this error.
rockchip-pinctrl pinctrl: pin gpio0-2 already requested by vcc-host-5v-regulator; cannot claim for vcc-host1-5v-regulator
  rockchip-pinctrl pinctrl: pin-2 (vcc-host1-5v-regulator) status -22
  rockchip-pinctrl pinctrl: could not request pin 2 (gpio0-2) from group usb20-host-drv  on device rockchip-pinctrl
  reg-fixed-voltage vcc-host1-5v-regulator: Error applying setting, reverse things back

Described here: [https://patchew.org/linux/20230421213841...@brun.one/](https://patchew.org/linux/20230421213841.3079632-1-lorenz@brun.one/)

**Comment by johang** [comment]
That patch was merged in kernel 6.4.
